client/client.go

```diff
@@ -552,9 +558,14 @@ func (c *Client) Do(req *http.Request) (*container.Container, *http.Response, er
 			if ok := c.backoff(attempts); !ok {
 				obj, err := container.ParseJSON(bodyBytes)
 				if err != nil {
-					log.Printf("[ERROR] Error occured while json parsing %+v with HTTP StatusCode 405, 500-504", err)
+					log.Printf("[ERROR] Error occured while json parsing: %+v", err)
```

Comment:
please keep the "with HTTP StatusCode 405, 500-504" part so we differentiate the log message from L544

Reply:
Sure - I can I can keep this part.